.github/workflows/codeql.yml

```
@@ -0,0 +1,86 @@
+          ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build project
+      run: npm run build
+
+    - name: Run lint
+      run: npm run lint
+      if: success() && steps.install.outputs.hasLintScript == 'true'
```

actionlint (1.7.7)

property "install" is not defined in object type {}

(expression)